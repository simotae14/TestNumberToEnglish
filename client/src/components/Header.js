import React from 'react';
import {
    AppBar,
    Toolbar,
    Typography,
} from '@material-ui/core';

const Header = () => {
    return (
        <AppBar position="static">
            <Toolbar>
                <Typography variant="title" color="inherit">
                    Number to English
                </Typography>
            </Toolbar>
        </AppBar>
    );
};

export default Header;