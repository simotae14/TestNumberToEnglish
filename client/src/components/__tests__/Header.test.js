import React from 'react';
import Header from '../Header';
import renderer from 'react-test-renderer';

describe('Header Component', () => {

    it('render correctly Header component', () => {
        const HeaderComponent = renderer.create(<Header />).toJSON();
        expect(HeaderComponent).toMatchSnapshot();
    });
});