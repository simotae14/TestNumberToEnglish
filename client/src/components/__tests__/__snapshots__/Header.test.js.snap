// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component render correctly Header component 1`] = `
<header
  className="MuiPaper-root-9 MuiPaper-elevation4-15 MuiAppBar-root-1 MuiAppBar-positionStatic-5 MuiAppBar-colorPrimary-7"
>
  <div
    className="MuiToolbar-root-36 MuiToolbar-gutters-37"
  >
    <h2
      className="MuiTypography-root-38 MuiTypography-title-44 MuiTypography-colorInherit-57"
    >
      Number to English
    </h2>
  </div>
</header>
`;
